src/main/java/com/dawn/jat/illuminati/post/controller/PostController.java

```diff
@@ -38,4 +41,26 @@
         }
         return new ResponseEntity<>(postEntity, HttpStatus.OK);
     }
-}
+
+    /**
+     * Gets list of Post Brief.
+     */
+    @GetMapping(value = "/list-post-brief")
```

Comment:
`@GetMapping(value = "/all-post/summary")`
change to this for better url 👯‍♂

Reply:
![image](https://user-images.githubusercontent.com/7389488/72857686-fe085d00-3cf0-11ea-94c9-c3e172ab0c5f.png)

Updated